### ISSUE TITLE
Export data in Tensorboard

### ISSUE THREAD

**Issue by McLawrence** [lanpa/tensorboard-pytorch#169]
I know it is possible to write the scalar values using `export_scalars_to_json()`. In Tensorboard, however, you can export scalars in the UI to JSON or CSV. Is there a possibility to do this in tensorboardX? If I haven't exported the scalars but need them from older runs, is there a possibility to extract the values from `events.out.tfevents....` file?

**Comment by lanpa** [comment]
Do you mean there is no button to export scalars to JSON or CSV in the tensorboard UI?

**Comment by McLawrence** [comment]
Correct, I wasn't sure if it is connected with TensorboardX or with my Tensorboard installation.

**Comment by lanpa** [comment]
Do you know there's a `Show data download links` on top left?
